lib/execution_engine2/sdk/EE2Status.py

```diff
@@ -560,26 +558,28 @@ def _send_exec_stats_to_catalog(self, job_id):
 
         self.sdkmr.get_catalog().log_exec_stats(log_exec_stats_params)
 
-    def abandon_children(self, parent_job_id, child_job_ids, as_admin=False) -> Dict:
-        if not parent_job_id:
-            raise ValueError("Please provide valid parent_job id")
+    def abandon_children(self, batch_id, child_job_ids, as_admin=False) -> Dict:
+        # Note this does not work for 'manual' batch jobs as the parent job is
+        # never updated with the child jobs. It will only work with batch jobs specifically
+        # created by the run_job_batch endpoint.
+        if not batch_id:
+            raise ValueError("Please provide valid batch_id")
         if not child_job_ids:
             raise ValueError("Please provide job_ids of children to abandon")
 
         job = self.sdkmr.get_job_with_permission(
-            parent_job_id, JobPermissions.WRITE, as_admin=as_admin
+            batch_id, JobPermissions.WRITE, as_admin=as_admin
         )  # type: Job
         for child_job_id in child_job_ids:
             if child_job_id not in job.child_jobs:
                 raise ChildrenNotFoundError(
                     f"Couldn't find {child_job_id} in {child_job_ids}"
                 )
 
-        with self.sdkmr.get_mongo_util().mongo_engine_connection():
-            job.update(pull_all__child_jobs=child_job_ids)
-            job.reload()
+        job.update(pull_all__child_jobs=child_job_ids)
+        job.reload()
 
-        return {"parent_job_id": parent_job_id, "child_jobs": job.child_jobs}
+        return {"parent_job_id": batch_id, "child_jobs": job.child_jobs}
```

Comment:
I think we wanted to rename `parent_job_id` here

Reply:
yup -- the narrative will be expecting this to be `batch_id`

Reply:
Fixed, good catch. I also changed `child_jobs` to `child_job_ids` since that's what the spec says it's supposed to be.